### DISCUSSION TITLE
English Language Leader

### DISCUSSION THREAD

**Discussion by MH-CODES** [OHF-Voice/intents#350] [Language Leaders]
Happy to put my name down for English Language Leader. Thought there could be one already.

Would love to contribute to this wonderful community.

**Comment by AalianKhan** [comment]
Hey!
Was looking over the response for english and noticed the cover area response is config is this
`cover_area: "Closed {{ slots.area }}"`
Doesn't this seem a bit odd? For example, when I want to close the blinds in the living room, it replies with closed living room
